src/main/java/com/minecolonies/coremod/entity/ai/citizen/lumberjack/Tree.java

```diff
@@ -143,7 +143,7 @@ else if (block instanceof BlockNewLog)
     public static boolean checkTree(@NotNull final IBlockAccess world, final BlockPos pos)
     {
         //Is the first block a log?
-        if (!world.getBlockState(pos).getBlock().isWood(world, pos))
+        if (!world.getBlockState(pos).getBlock().isWood(world, pos) && !world.getBlockState(pos).getMaterial().equals(Material.CLAY) || world.getBlockState(pos).getBlock() == Blocks.CLAY)
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Add parentheses to make the operator precedence explicit. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=squid%3AS864)
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png 'Severity: MINOR') Split this 187 characters long line (which is greater than 180 authorized). [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=squid%3AS00103)